platform/darwin/src/MGLGeoJSONSource.h

```diff
@@ -59,6 +59,15 @@ extern const MGLGeoJSONSourceOption MGLGeoJSONSourceOptionSimplificationToleranc
 /**
  A GeoJSON source.
 
+ ### Example ###
+ 
+ ```swift
+ var coordinates: [CLLocationCoordinate2D] = []
+ let polyline = MGLPolylineFeature(coordinates: &coordinates, count: UInt(coordinates.count))
+ let source = MGLGeoJSONSource(identifier: "lines", features: [polyline], options: nil)
```

Comment:
#7145 renamed this initializer to `MGLGeoJSONSource(identifier:shape:options:)`, and #7334 would go a step further, renaming the class to MGLShapeSource. It still makes sense to demonstrate creating the source with a feature rather than an unadorned shape.